frontend/src/components/navigation/Header.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+/*import React from 'react';
 import styled from 'styled-components';
 import { NavbarLink } from '../core/Link';
 
```

Comment:
As mentioned above, un-comment this whole section, as we'll be using Styled Components in this project. There are a fair number of examples in this project of how they work, I suggest experimenting around with them. Feel free to ask any questions 👍